nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -176,6 +176,10 @@
   instead of the python tester launcher. You can still refer to the python
   launcher via `python3Packages.toPythonApplication python3Packages.playwright`
 
+- Using `==` or `!=` to compare elaborated platform attrsets, such as `stdenv.hostPlatform` and `stdenv.buildPlatform`, is no longer supported and may produce a warning. In a future version, this warning will be replaced with an error. Instead, you should use the `canExecute`, `equals`, or `notEquals` functions included with platform attrsets or use `lib.systems.equals`.
+
+  - E.g. replace `hostPlatform == buildPlatform` with `buildPlatform.canExecute hostPlatform`.
```

Comment:
Added a release note to the "Backward Incompatibilities" section.

Alternative wording suggestions welcome :upside_down_face:

Reply:
Hm, should this go in [`doc/release-notes`](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2505.section.md) or [`nixos/doc/manual/release-notes`](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md)? Maybe both?

Reply:
Thanks. Moved :+1: